modules/rosa-planning-considerations.adoc

```diff
@@ -1,22 +1,25 @@
 
 // Module included in the following assemblies:
 //
-// rosa_planning/rosa-planning-environment.adoc
+// rosa_planning/rosa-limits-scalability.adoc
 
-[id="initial-planning-considerations_{context}"]
-= Initial planning considerations
+[id="control-plane-and-infra-node-sizing-and-scaling_{context}"]
+= Control plane and infrastructure node sizing and scaling
 
-Consider the following tested object maximums when you plan your {product-title} cluster.
+When you install a {product-title} (ROSA) cluster, the number of required control plane and infrastructure nodes is automatically determined by the compute node count.
```

Comment:
```suggestion
When you install a {product-title} (ROSA) cluster, the sizing of the control plane and infrastructure nodes are automatically determined by the compute node count.
```

Reply:
Thank you for highlighting this. Resolved.